clinica/pipelines/space_required_by_pipeline.csv

```diff
@@ -1,17 +1,14 @@
 pipeline_name;space_caps;space_wd
-T1FreeSurfer;1G;100K
-T1FreeSurferLongitudinal;400M;400M
-T1VolumeDartel2MNI;30M;150M
-T1VolumeNewTemplate;300M;275M
-T1VolumeTissueSegmentation;35M;147M
-T1VolumeCreateDartel;130M;140M
-T1VolumeExistingTemplate;55M;320M
-T1VolumeParcellation;760K;520K
-DwiPreprocessingUsingT1;150M;3G
-DwiPreprocessingUsingPhaseDiffFieldmap;250M;6G
-DwiConnectome;1G;2G
-DwiDti;400M;700M
-fMRIPreprocessing;300M;7G
-PETVolume;55M;245M
-PetSurface;25M;1G
-SpatialSVM;5M;700M
\ No newline at end of file
+t1-freesurfer;1G;100K
+t1-volume-dartel2mni;30M;150M
+t1-volume-tissue-segmentation;35M;147M
+t1-volume-create-dartel;130M;140M
+t1-volume-parcellation;760K;520K
+dwi-preprocessing-using-t1;150M;3G
+dwi-preprocessing-using-fieldmap;250M;6G
+dwi-connectome;1G;2G
+dwi-dti;400M;700M
+fmri-preprocessing;300M;7G
+pet-volume;55M;245M
+pet-surface;25M;1G
+machine-learning-prepare-spatial-svm;5M;700M
```

Comment:
Actually, I understood that you modified this file because when you instantiate pipeline in CLI, you set the pipeline name to be self.name which is the name of the CLI. By doing that, we loose the ability to have the `check_size` if the pipeline is not launched through the CLI (because the class name would be different)

Reply:
See 9fa8477 for improvements. This should avoid the situation you are mentionning and enforce pipelines to stay in their respective folders. :)

Reply:
Ok perfect